src/mqtt_client/mod.rs

```diff
@@ -318,10 +318,8 @@ where
     fn handle_subscription(&mut self) {
         log::info!("MQTT connected, subscribing to settings");
 
-        // Note(unwrap): We construct a string with two more characters than the prefix
-        // structure, so we are guaranteed to have space for storage.
         let mut settings_topic = self.prefix.clone();
-        settings_topic.push_str("/#").unwrap();
+        settings_topic.push_str("/settings/#").unwrap();
```

Comment:
I'm confused. The subscription options should have prevented the alive and telemetry messages

Reply:
Yeah, we expect to get the settings prefix and subscribe at the upper level because we parse the commands get/set that occur below the settings tree

https://github.com/quartiq/miniconf/blob/8be449dcd9be7c838720c21223dac0841595dd26/src/mqtt_client/mod.rs#L97

Reply:
Not telemetry. That's a different client. But the alive, yes. Maybe.

Reply:
Ah right.